### DISCUSSION TITLE
How to use with wsl?

### DISCUSSION THREAD

**Discussion by z-plugin** [zh-lx/code-inspector#235] [Q&A] [answered]
I use vscode with wsl and code-inspector-plugin can't open the vscode. How should I do?

**Comment by zh-lx** [comment, marked as answer]
You can exec `which code` on terminal in wsl, and it will show a result like: `/home/xxx/.vscode-server/bin/dc96b837cf6bb4af9cd736aa3af08cf8279f7685/bin/remote-cli/code`.

Copy it and add to `.env.local`:
```
CODE_EDITPR=/home/xxx/.vscode-server/bin/dc96b837cf6bb4af9cd736aa3af08cf8279f7685/bin/remote-cli/code
```

**Reply by z-plugin** [reply]
Thanks! It works!
